docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -9,6 +9,8 @@ The default ASP.NET Core port configured in .NET container images has been updat
 
 We also added the new `ASPNETCORE_HTTP_PORTS` environment variable as a simpler alternative to `ASPNETCORE_URLS`. The new variable expects a semicolon delimited list of ports numbers, while the older variable expects a more complicated syntax.
 
+Apps build using the older [`WebHost.CreateDefaultBuilder`](https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.webhost.createdefaultbuilder) API won't respect the new `ASPNETCORE_HTTP_PORTS` environment variable and will therefore switch to using a default port of 5000, now that `ASPNETCORE_URLS` is no longer set automatically.
```

Comment:
This is port 5000 on `localhost`, which means the port is no longer accessible outside the container.

A user can change the code of their app, or they can set `ASPNETCORE_URLS` to `http://*:8080` to match the port exposed by the base .NET 8 base image.